### PR TITLE
Bug 1128390 - Confirm removal and donwload error toast for dict download

### PR DESCRIPTION
**Pull request by timdream**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c37222446da8d50a7376c7ab3d763fb4f9f0ec14

File: apps/keyboard/js/settings/layout_item_list_view.js [hunk 1]
```diff
@@ -4,6 +4,144 @@
 
 (function(exports) {
 
+// This is a simply wrapper wraps <gaia-toast> into our view management.
```
Comment (RudyLu) [inline apps/keyboard/js/settings/layout_item_list_view.js]: -> simple wrapper that wraps...

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=57077f4266835458ce9aca01c419a2925fdf0c67

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7ff5ea6c6c0e1454f55a14db1376f1f8a482a70c

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#qEU8dDPgQdS1gWIsVqNiRQ

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#BfkKE-qtSTuF7aJFt-32nA

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9e4ff5d011f2d57e23240c49adc9aead2da30d9f

**Pull request merged**